### ISSUE TITLE
User Search Clarification

### ISSUE THREAD

**Issue by russdeffner** [hotosm/tasking-manager#1346]
When you are searching for users, you have the option to search users by role or by mapping level. What is confusing is you can actually filter on both, but there is no visual indicator that that is what is happening. i.e. if I filter for validators, then click beginner mappers, the button for validators turns off making me think I'm seeing all beginners, yet the list is still filtered for validators. Which is exactly what I want, just had to verify that's actually what I was getting. 

Long story short: the sort by role and level radio buttons need to better indicate what is displayed.

**Comment by russdeffner** [comment]
A couple of suggestions to resolve this:
* Make both filter have a 'permanent state', i.e. each menu should always show something (All by default)
* Expand the "Total number of users:" to be more descriptive; i.e. 'Total number of users in the X role/Y level'

Somewhat like shopping Amazon or Kayak, etc. lots of independent filters with a description like 'you're looking at maps in the mapping department under $10' or something along those lines :)

**Comment by russdeffner** [comment]
Closing as this has been implemented in TM4
